e2e/bitrise.yml

```diff
@@ -19,6 +19,37 @@ app:
 
 workflows:
 
+  test_visionOS:
+    description: visionOS SDK requires Xcode 15.2+
+    steps:
+    - bitrise-run:
+        title: Run visionOS test if current stack supports it
+        run_if: |-
+            {{ and (ne (getenv "BITRISEIO_STACK_ID") "osx-xcode-15.1.x") (ne (getenv "BITRISEIO_STACK_ID") "osx-xcode-15.0.x") (ne (getenv "BITRISEIO_STACK_ID") "osx-xcode-14.3.x-ventura") (ne (getenv "BITRISEIO_STACK_ID") "osx-xcode-14.2.x-ventura") (ne (getenv "BITRISEIO_STACK_ID") "osx-xcode-14.1.x-ventura") (ne (getenv "BITRISEIO_STACK_ID") "osx-xcode-14.0.x-ventura") }}
```

Comment:
This ugly expression is going to get better over time (we'll remove Xcode 14.0 in 4 weeks, for example).

I also tried to make it easier to read by adding `{{ with $stack := getenv "BITRISEIO_STACK_ID" }} ... {{ end }}`, but it breaks the expression, it seems like it always returns a truthy value.

Reply:
Also, too bad the templating doesn't have a `strings.HasPrefix("osx-xcode-14")` equivalent :(

Reply:
BTW there is envcontain https://github.com/bitrise-io/bitrise/blob/83e3e4058ae1edc4ee4a3fb30e6c48aaea0dc562/bitrise/template_utils_test.go#L95